### PR TITLE
docs: better README

### LINKED ISSUE #367: Better README
# Bounty

Current readme is reported to be outdated and overwhelming for newbies: too many details, no quick start path. Suggesting to restructure it.

## Scope

- Start with quick and easy overview:
   - what it is for
   - birds-eye architectural view (node + root chain), so you can get a sense where the node fits into the whole picture
   - what to do after the start (JSON RPC)
- Absolute minimum actions to spin up a node for testnet/mainnet (aka "Quick Start")
- Advanced configuration (e.g. optional CLI args, logging, config flags etc). Consider moving advanced things into documentation and reference from README (like Tendermint guys do)
- How to contribute:
   - reference CONTRIBUTION guide from meta repo (the guide itself is out of scope here)
   - mention or reference bounty program and point of contact (Slack, Dev Link)

## Deliverables

- updated README

## Gain for the project

- easier onboarding of core devs

### Publicly visible delivery

- updated README

## Roles
bounty gardener: @troggy / 40 DAI
bounty worker: @troggy  / 80%
bounty reviewer: @eswarasai  / 20%

## Funded by

Dev Circle

### PR DESCRIPTION
**Pull request by troggy**
- added architectural overview
- restructured the file with focus on general public
- development and advanced configuration moved out to a separate file and docs.leapdao.org

Resolves #367

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+* [Debug via Blink Developer Tools / node-inspector](#debug-via-blink-developer-tools--node-inspector)
+* [Config file options](#config-file-options)
+* [Feature flags](#feature-flags)
+  * [How to add a new flag](#how-to-add-a-new-flag)
+  * [How to configure feature flags](#how-to-configure-feature-flags)
+* [Run in the cloud](#run-in-the-cloud)
+  * [Initial setup (one time per developer machine)](#initial-setup-one-time-per-developer-machine)
+  * [Create/update AWS EC2 cluster](#createupdate-aws-ec2-cluster)
+* [Dive in development](#dive-in-development)
+
```
Comment (eswarasai) [inline DEVELOPMENT.md]: @troggy - Might want to start with clone the repository? Just to be explicit is all.

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+To connect to the testnet
+
+```sh
+DEBUG=tendermint,leap-node* node index.js --config=https://testnet-node.leapdao.org
+```
+
+## Run local network
+
+Use [leap-sandox](leapdao/leap-sandbox). It is the easiest way to start local leap network at the moment.
```
Comment (eswarasai) [inline DEVELOPMENT.md]: @troggy - Update the URL `https://github.com/leapdao/leap-sandbox`?

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  "bridgeAddr": "0x7b8342412883f4b34f335d4e1391ec190eb887ca",
+  "operatorAddr": "0x9b83018de826c0343af6e682c24e7c91a421755c",
+  "exitHandlerAddr": "0x0b32eb1aaa9b0804852f4fe1b2e6100edb4533d8",
+  "rootNetwork": "https://rinkeby.infura.io",
+  "rootNetworkId": 4,
+  "network": "leap-testnet-gamma",
+  "networkId": 1341,
+```
+
+* `bridgeAddr` — leap [Bridge](https://github.com/leapdao/leap-contracts) contract address
```
Comment (eswarasai) [inline DEVELOPMENT.md]: @troggy - Link to the contract directly? -- `https://github.com/leapdao/leap-contracts/blob/master/contracts/Bridge.sol`

Recommend doing the same for below contracts as well. Easy to refer.

Comment (troggy) [reply DEVELOPMENT.md]: yeah.. these all are good points. I mostly didn't touch Development section at all, just moved out from README

**Comment by troggy** [comment]
@eswarasai addressed your comments

**Pull request merged**